ingestion-beam/Dockerfile

```diff
@@ -0,0 +1,20 @@
+
+FROM maven:3-jdk-8 AS build
+WORKDIR /app/ingestion-beam
+COPY pom.xml /app/pom.xml
+COPY ingestion-beam/pom.xml /app/ingestion-beam/pom.xml
+COPY checkstyle /app/checkstyle
+COPY ingestion-beam/checkstyle /app/ingestion-sink/checkstyle
+RUN mvn prepare-package
+COPY ingestion-core/src /app/ingestion-core/src
+COPY ingestion-beam/src /app/ingestion-beam/src
+RUN mvn package -Dmaven.test.skip=true
+
+FROM gcr.io/dataflow-templates-base/java8-template-launcher-base:latest
```

Comment:
```suggestion
# This is the base image for the final image we're building.
FROM gcr.io/dataflow-templates-base/java8-template-launcher-base:20210120_RC00
```

It is specifically recommended not to choose latest: https://cloud.google.com/dataflow/docs/guides/templates/configuring-flex-templates